docs/src/pages/components/data-grid/columns/ColumnSelectorGrid.tsx

```diff
@@ -11,7 +11,13 @@ export default function ColumnSelectorGrid() {
 
   return (
     <div style={{ height: 400, width: '100%' }}>
-      <DataGrid {...data} showToolbar />
+      <DataGrid
+        {...data}
+        showToolbar
+        components={{
+          Toolbar: GridToolbar,
+        }}
+      />
```

Comment:
A side note, now I understand why I got confused by the presence of the `showToolbar` prop. I wasn't expecting any changes in this demo. It looks like we anticipate the next change. But it's actually unrelated to the value proposition of fixing #946. It's fine for this time, I only wanted to clarify it.

Reply:
Ok, yes I could have left the default toolbar still there but instead, I removed it, I figured it is ok since that was what we discussed when we were refining the ticket. I can reintroduce it if we want to be as granular as possible and remove it in the other PR.